sdk/go/outbound_redis/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: generate
```

Comment:
Could there be a single Makefile for the Go SDK that handles both the bindings generation and building / testing the SDK?

Reply:
Yes but I want to give testing and code management more though. I don't want it to hold up this PR.